source/handshake.c

```diff
@@ -396,13 +396,14 @@ int main(int argc, char * const argv[])
         goto err;
     }
 
-    ttime = times[0];
-    for (i = 1; i < threadcount; i++)
-        ttime = ossl_time_add(ttime, times[i]);
+    avcalltime = (double)0;
+    persec += (double)0;
+    for (i = 0; i < threadcount; i++) {
+        avcalltime += ((double)ossl_time2ticks(times[i]) / (num_calls / threadcount)) / (double)OSSL_TIME_US;
+        persec += (((num_calls * OSSL_TIME_SECOND) / (double)threadcount) / (double)ossl_time2ticks(times[i]));
```

Comment:
I do not think this is a correct change. Either we want to compute the wall clock time of all the threads spent in the computation or we want to compute how much time it took to perform the num_calls operations regardless of the number of threads involved.  This is something in the middle that has no real meaning IMO.

Furthermore we will loose consistency in the graphs tracking the library changes over time.

I instead propose to use the existing persec number for comparison across number of threads.

Reply:
agreed, makes sense.  I'll close this in favor of a new pr that addresses all tests shortly.